test/integration/middleware_test.rb

```diff
@@ -31,7 +31,6 @@ class MiddlewareIntegrationTest < ActionDispatch::IntegrationTest
     test 'it is added Content-Security-Policy on welcome pages' do
       Domain.stubs(:any?).returns(false)
       visit '/domains'
-      assert page.has_content? 'Learn more about this in the documentation.'
```

Comment:
note to any other reviewer - this assertion shouldn't have been part of this test in the first place.